persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java

```diff
@@ -70,11 +70,9 @@ public void testXmlConfig() throws IOException {
 
       Cache<String, String> cache = cacheManager.getCache("testCache");
 
-      cache.put("hello", "there 60 xml");
+      cache.put("hello", "there 52 xml");
       cache.stop();
       cacheManager.stop();
-
-      Util.recursiveFileRemove("/tmp/leveldb/60");
```

Comment:
Yes we do in the `@AfterClass` method: https://github.com/slaskawi/infinispan/blob/40c789baabdd408388bb1c889b7420d14702a779/persistence/leveldb/src/test/java/org/infinispan/persistence/leveldb/config/ConfigurationTest.java#L35-L38